libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -890,6 +891,7 @@ static int vk_init(struct gpu_ctx *s)
     s->limits.max_texture_dimension_2d           = limits->maxImageDimension2D;
     s->limits.max_texture_dimension_3d           = limits->maxImageDimension3D;
     s->limits.max_texture_dimension_cube         = limits->maxImageDimensionCube;
+    s->limits.max_texture_array_layers           = limits->maxImageArrayLayers;
```

Comment:
the full list is not in alphabetic order but for consistency you could keep it at least for the `max_texture` subset

Reply:
I choose this order to match VkPhysicalDeviceLimits, see: https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceLimits.html.